### DISCUSSION TITLE
Dashy consistently shuts itself off after an hour or two

### DISCUSSION THREAD

**Discussion by jeremytodd1** [Lissy93/dashy#630] [Help and Support] [answered]
I have around 25 containers and all of them work just fine and are stable. The only one I'm having issue with is the container for Dashy (Dashy is a dashboard). After a little bit of time the container just shuts itself off. I can restart it and it'll boot back up just fine, but it'll again just shut back down after a while. I've also tried recreating the container and the same thing occurs.

Anyone know what this means and how I can fix this?

If it makes a difference I'm using Unraid.

I click the "Logs" button for the container and these are the last few lines in the log:
```dist/css/chunk-b8d94e6a.0e433876.css 0.00 KiB 0.02 KiB

Images and other types of assets omitted.

DONE Build complete. Watching for changes...
error Command failed with signal "SIGKILL".

info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
ERROR: "start" exited with 1.

error Command failed with exit code 1.

info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.```

**Comment by lissy93** [comment, marked as answer]
> See: **[Docs --> Troubleshooting --> Command failed with signal "SIGKILL"](https://github.com/Lissy93/dashy/blob/master/docs/troubleshooting.md#command-failed-with-signal-sigkill)**

This is common if you've put a tight memory restriction in place. There's a spike in usage when the container builds, so you'll see that error if there's not enough memory. It's easy to fix with `--memory` (or alternate [runtime option](https://docs.docker.com/config/containers/resource_constraints/)). There's more info about it in the troubleshooting docs.

**Reply by lissy93** [reply]
And glad you specified that _"Dashy is a dashboard"_ - I wouldn't have been sure what Dashy was otherwise 😉🤣

**Reply by jeremytodd1** [reply]
Lol, I pasted this from a reddit thread I made. The "dashy is a dashboard" is a bit unnecessary here for sure, lol. I forgot to take that out.

To install Dashy I just used Unraid's Community Apps store. I didn't change anything as far as memory management goes.

Here is an image of what settings comes by default for the container:
https://i.imgur.com/bvoVjnQ.png

It's also odd, as my server has 32GBs of memory and it's only using around 62% of that.
